libs/prisma-inflector/src/rules.rs

```diff
@@ -8,7 +8,7 @@ pub trait Pluralize {
 #[derive(Debug)]
 pub enum Rule {
     Category(CategoryRule),
-    Regex(RegexRule),
```

Comment:
That one is unexpected - doesn't that have the side effect of always heap allocating?

Reply:
We talked about this on Slack -- the answer is "yes", but the overhead isn't that big, and it can significantly speed things up for the other branches.